### ISSUE TITLE
OPNsense 23.7-amd64 / OpenVPN new Instances pages creation doesn't save 'Enforce local group' setting

### ISSUE THREAD

**Issue by mm-secab** [opnsense/core#6692]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Describe the bug**

When creating or editing an OpenVPN instance from the new instances pages, the "enforce local group" selection is not saved.

No error message displayed.

**To Reproduce**

1. Go to VPN/Instances [new];
2. Create or edit an instance;
3. Scroll down to 'Enforce local group' and select an existing group;
4. Click on 'Save';
5. Click on 'Apply';
6. Open the instance created or edited. The 'Enforce local group' returned to 'None'.

**Expected behavior**

Saving the instance must save the value indicated in 'Enforce local group'.

**Describe alternatives you considered**

n/a

**Screenshots**

n/a

**Relevant log files**

n/a

**Additional context**

n/a

**Environment**

OPNsense 23.7-amd64.

**Comment by fichtner** [comment]
It looks like the form/JS doesn't render... the value seems to be there and selected in the API response:

```
        "local_group": {
            "": {
                "value": "none",
                "selected": false
            },
            "1999": {
                "value": "admins",
                "selected": 1
            }
        },
```

**Comment by fichtner** [comment]
Fixed via 68ae7bd, thanks a lot for the quick report!

    # opnsense-patch 68ae7bd


Cheers,
Franco
